### DISCUSSION TITLE
Error launching app -> Unable to find Electron app at .../drawio-desktop-13.3.1/drawio/src/main/webapp

### DISCUSSION THREAD

**Discussion by CholoTook** [npm/rfcs#172] [General]
Not sure what this means...

My error is pasted in full here:
https://termbin.com/1loqj

Any feedback welcome.

**Comment by ljharb** [comment]
You also filed this here: https://github.com/npm/cli/issues/1461

> `npm start` is part of a local application's package.json, and doesn't have anything to do with npm. you'll need to file an issue on that project.

**Reply by noofh-22** [reply]
hi

#438

**Reply by devomor** [reply]
hi

**Reply by warushayohan** [reply]
hi

**Reply by GatoArtStudio** [reply]
hi

**Reply by mauladiarif231** [reply]
hi

**Reply by chienlb** [reply]
hi

**Comment by CholoTook** [comment]
Sorry, outside of IRC I get confused.

**Reply by JhonnyLusonode** [reply]
> Sorry, outside of IRC I get confused.

i had the same issue 😂

**Comment by mithatyagmuroglu** [comment]
You also filed this here: https://github.com/npm/cli/issues/1461

**Reply by warushayohan** [reply]
hey

**Reply by mauladiarif231** [reply]
test

**Reply by chienlb** [reply]
hi

**Comment by akayush1108** [comment]
I've some issues in my repo can u pls help me to figure it out?

**Reply by shaswat0207** [reply]
Yes dude.

**Reply by akayush1108** [reply]
😊

**Reply by shaswat0207** [reply]
👍

**Reply by sujit-al1809** [reply]
Yes Please

**Comment by Md-Danish-au45** [comment]
nice...

**Reply by Uliana200407** [reply]
Response #1

**Comment by Pathum-lakshan** [comment]
I've some issues in my repo can u pls help me to figure it out?

**Reply by PremKaneriya** [reply]
response

**Reply by mauladiarif231** [reply]
test

**Comment by Pathum-lakshan** [comment]
My error is pasted in full here:
https://termbin.com/1loqj

Any feedback welcome.

**Comment by soufiiyane** [comment]
please i've got same error ! anyone care to help !

**Reply by sujit-al1809** [reply]
Please tell

**Comment by wesleytodd** [comment]
For those reading this, these discussions are not intended to be a support forum. You should probably post these on StackOverflow or Reddit?

Anyone have the ability to close this and lock the discussion?

**Reply by wesleytodd** [reply]
@wraithgar maybe? Sorry, I would be happy if you gave me a triager role to help clean these up.

**Comment by Uliana200407** [comment]
Response#1

**Reply by Uliana200407** [reply]
Respose#1
